blog/2021-07-08-xmr-btc-project-hibernation.md

```diff
@@ -0,0 +1,93 @@
+The protocol in use here is the [libp2p rendezvous protocol](https://github.com/libp2p/specs/tree/master/rendezvous).
+While still in its infancy, it is serving us well to deliver a working version of maker discovery.
+Once connected to the rendezvous point, the CLI requests a list of _registrations_ for a predetermined namespace.
+An ASB provider can [configure](https://github.com/comit-network/xmr-btc-swap/tree/master/docs/asb#asb-discovery) their instance to automatically publish such a registration on startup.
+
+But wait, did we say the tool is feature-complete?
+Not quite unfortunately.
+
+We would really like to offer buying AND selling for the user but this is blocked by transaction chaining / pre-signing landing in Monero.
+You can learn more on that topic in [this](/blog/2021/07/02/transaction-presigning) blogpost.
+
+In summary, we have pretty much delivered exactly what we wanted:
+
+A somewhat usable tool that allows users to trustlessly trade the XMR/BTC pair.
+
+## What's next
+
+We are still very keen on offering the missing feature of selling XMR for BTC.
+Outside of that, we also had some ideas on what _could_ be built on top of or with our tool:
+
+- Swaps completely in the browser:
+  Rust is a great language for targeting WASM.
+  Compiling everything to WASM would enable swaps completely within the browser, no download necessary.
+  The major obstacle here is our dependency on `monero-wallet-rpc` which we currently start transparently in the background.
+  This will either need to be replaced by something like [monero-javascript](https://github.com/monero-ecosystem/monero-javascript) or a pure Rust implementation of a Monero wallet.
+  Both approaches are not exactly a weekend project but well within reach for anyone that knows their way around Rust.
+- A GUI for the swap CLI:
+  At the moment, all that is offered to the user is a CLI.
+  Whilst sufficient for expert users and to showcase that it works, a proper GUI would be a lot more user-friendly.
+  Fortunately, the CLI is completely non-interactive.
+  If anyone wants to build a GUI, the recommended way would be to start the CLI for the user in the background and process the logs delivered on stderr.
+  An adventurous developer could also try to build a GUI directly in Rust and integrate on a library level.
+- A mobile app.
+  We have very little experience in our team when it comes to writing mobile apps which is why we didn't focus on this at all.
+  For all we know, Rust can target Android as well as iOS which means there should be a way of getting this onto mobile!
+
+This list is by no means exhaustive and we would love for others to jump onto some of these ideas.
+Whilst we will not be pushing any of those forward ourselves, we are more than happy to assist by participating and steering [discussions](https://github.com/comit-network/xmr-btc-swap/discussions) and reviewing pull requests.
```

Comment:
You could thank the community for feedback and contributions and maybe even highlight the contributions we got.

Reply:
Nice idea, should do that yes!